qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -608,7 +608,7 @@
     <item>
      <widget class="QComboBox" name="comboBox_5">
       <property name="sizePolicy">
-       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
```

Comment:
The size is reasonable there - if a little over large - but the change is fine for now then, I don't want to nitpick at this stage.

What I'm unclear on though is if there are no upper bounds on the size of the box, will it stay a reasonable size under all resizing or other contortions. I think the overall toolbar is (or should be) set to be as big as necessary but no more, since some controls get hidden/shown and it should resize to fit. So if there's a control inside it that can resize as large as it likes I'm not sure how those end up settling.